metricbeat/module/docker/config.go

```diff
@@ -29,13 +30,15 @@ func DefaultConfig() Config {
 	}
 }
 
+// TLSConfig contains TLS config
```

Comment:
How about: `TLSConfig contains TLS settings required to connect to the docker daemon via TCP` ?